src/System.Net.Sockets.Legacy/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1349,32 +1350,35 @@ public void Socket_ReceiveFromDnsEndPoint_Throws()
         }
 
         [Fact]
-        [ActiveIssue(4004, PlatformID.OSX)]
+        [PlatformSpecific(~PlatformID.OSX)]
         public void ReceiveFromV4BoundToSpecificV4_Success()
         {
             ReceiveFrom_Helper(IPAddress.Loopback, IPAddress.Loopback);
         }
 
         [Fact]
-        [ActiveIssue(4004, PlatformID.OSX)]
+        [PlatformSpecific(~PlatformID.OSX)]
         public void ReceiveFromV4BoundToAnyV4_Success()
         {
             ReceiveFrom_Helper(IPAddress.Any, IPAddress.Loopback);
         }
 
         [Fact]
+        [PlatformSpecific(~PlatformID.OSX)]
```

Comment:
The tests that were using only IPv6 addresses were passing.  We could special-case this, but it would mean that whether you got a PNSE was determined by what your clients sent to you, which seems worse than just consistently failing.